lib/core/rancher/step.sh

```diff
@@ -49,19 +49,6 @@ function ih::setup::core.rancher::test() {
     return 1
   fi
 
-  # Use vz (requires macOS >=13.3) instead of qemu on M3 macs to resolve issues.
```

Comment:
Do you need to undo anything for people who have this set up? Or will the new Rancher just ignore it?

Reply:
I think both options work and qemu actually uses virtiofs under the hood, so the people who have it set up like this shouldn't have problems. 

I was planning to test with @chris-fitzpatrick-ih who has an m3 mac and had an issue with this recently: https://ih-epdd.slack.com/archives/C03GXCDA48Y/p1707777210265139?thread_ts=1705416222.093629&cid=C03GXCDA48Y